src/main/java/net/minecraftforge/common/Tags.java

```diff
@@ -36,15 +39,35 @@ private static void init(){}
         public static final TagKey<Block> BARRELS = tag("barrels");
         public static final TagKey<Block> BARRELS_WOODEN = tag("barrels/wooden");
         public static final TagKey<Block> BOOKSHELVES = tag("bookshelves");
+        public static final TagKey<Block> BUDDING_BLOCKS = tag("budding_blocks");
+        public static final TagKey<Block> BUDS = tag("buds");
         public static final TagKey<Block> CHESTS = tag("chests");
         public static final TagKey<Block> CHESTS_ENDER = tag("chests/ender");
         public static final TagKey<Block> CHESTS_TRAPPED = tag("chests/trapped");
         public static final TagKey<Block> CHESTS_WOODEN = tag("chests/wooden");
+        public static final TagKey<Block> CLUSTERS = tag("clusters");
         public static final TagKey<Block> COBBLESTONE = tag("cobblestone");
         public static final TagKey<Block> COBBLESTONE_NORMAL = tag("cobblestone/normal");
         public static final TagKey<Block> COBBLESTONE_INFESTED = tag("cobblestone/infested");
         public static final TagKey<Block> COBBLESTONE_MOSSY = tag("cobblestone/mossy");
         public static final TagKey<Block> COBBLESTONE_DEEPSLATE = tag("cobblestone/deepslate");
+        public static final TagKey<Block> DYED_BLOCKS = tag("dyed_blocks");
+        public static final TagKey<Block> DYED_BLOCKS_BLACK = tag("dyed_blocks/black");
+        public static final TagKey<Block> DYED_BLOCKS_BLUE = tag("dyed_blocks/blue");
+        public static final TagKey<Block> DYED_BLOCKS_BROWN = tag("dyed_blocks/brown");
+        public static final TagKey<Block> DYED_BLOCKS_CYAN = tag("dyed_blocks/cyan");
+        public static final TagKey<Block> DYED_BLOCKS_GRAY = tag("dyed_blocks/gray");
+        public static final TagKey<Block> DYED_BLOCKS_GREEN = tag("dyed_blocks/green");
+        public static final TagKey<Block> DYED_BLOCKS_LIGHT_BLUE = tag("dyed_blocks/light_blue");
+        public static final TagKey<Block> DYED_BLOCKS_LIGHT_GRAY = tag("dyed_blocks/light_gray");
+        public static final TagKey<Block> DYED_BLOCKS_LIME = tag("dyed_blocks/lime");
+        public static final TagKey<Block> DYED_BLOCKS_MAGENTA = tag("dyed_blocks/magenta");
+        public static final TagKey<Block> DYED_BLOCKS_ORANGE = tag("dyed_blocks/orange");
+        public static final TagKey<Block> DYED_BLOCKS_PINK = tag("dyed_blocks/pink");
+        public static final TagKey<Block> DYED_BLOCKS_PURPLE = tag("dyed_blocks/purple");
+        public static final TagKey<Block> DYED_BLOCKS_RED = tag("dyed_blocks/red");
+        public static final TagKey<Block> DYED_BLOCKS_WHITE = tag("dyed_blocks/white");
+        public static final TagKey<Block> DYED_BLOCKS_YELLOW = tag("dyed_blocks/yellow");
         public static final TagKey<Block> END_STONES = tag("end_stones");
         public static final TagKey<Block> ENDERMAN_PLACE_ON_BLACKLIST = tag("enderman_place_on_blacklist");
```

Comment:
This tag has functional behavior rather than being for commonality between mods, it should remain in `forge` or `neoforge`.